contrast-rest-dotnet/TeamServerClient.cs

```diff
@@ -436,6 +436,36 @@ public Organization GetDefaultOrganization()
             return response.organization;
         }
 
+        /// <summary>
+        /// Get a organization information based on its UUID.
+        /// </summary>
+        /// <param name="organizationId">Organization UUID</param>
+        /// <returns>A object that contains basic organization data</returns>
+        public OrganizationManagedResponse GetOrganizationInfo(string organizationId)
+        {
+            return GetOrganizationInfo(organizationId, null);
+        }
+
+        /// <summary>
+        /// Get a organization information based on its UUID.
+        /// </summary>
+        /// <param name="organizationId">Organization UUID</param>
+        /// <param name="expand">Load additional data</param>
+        /// <returns>A object that contains basic organization data</returns>
+        public OrganizationManagedResponse GetOrganizationInfo(string organizationId, List<OrganizationExpandValues> expand)
+        {
+            string endpoint = String.Format(NgEndpoints.ORGANIZATION_INFORMATION, organizationId);
+            if (expand?.Count > 0)
+            {
+                endpoint += "?expand=";
+                for (int i = 0; i < expand.Count; i++)
+                {
+                    endpoint += i > 0 ? "," + expand[i].ToString() : expand[i].ToString();
```

Comment:
This logic seems backwards.  Also hard to read.
I would use string.Join for this.  
endpoint = "?expand=" + string.Join(",", expand);

Reply:
Done. I was wondering if there was something like the StringUtils.join function and completely forgot that most of the equivalents are already in the String class.